virl2_client/models/node.py

```diff
@@ -443,6 +446,19 @@ def config(self, value: str) -> None:
         )
         self.configuration = value
 
+    @property
+    def parameters(self) -> dict:
+        self.lab.sync_topology_if_outdated()
+        return self._parameters
+
+    def update_parameters(self, new_params: dict) -> None:
```

Comment:
can this be a setter?

Reply:
I find the setter a little confusing because it's more of a patch than a set. User can update only a few parameters here, not necessarily all of them to rewrite the whole dict.

Reply:
I agree that it's not a good fit for a setter.